README.md

```diff
@@ -1,4 +1,6 @@
 This will contain items for the Organization level documentation.
 
 
-Additionally this readme is here to allow branches to be created from the empty repo.
+This repo contains items for the Organization level documentation.
+
+As well as it contains some documentation, even if its planned to move some to `pulsar-edit.github.io`
```

Comment:
I'd add in our image parts here once we get my outstanding PR in pulsar-edit/pulsar-edit.github.io for the doc-fixes branch merged. There are some blocking changes to that that will have to be addressed before that point however.

Reply:
We absolutely could. But should probably wait until thats done over there, then we can make another PR here, if that sounds alright with you?